src/Gui/propertyeditor/PropertyItem.cpp

```diff
@@ -2215,6 +2215,9 @@ void PropertyStringListItem::setValue(const QVariant& value)
 
         std::string pystr = Base::Tools::escapedUnicodeFromUtf8(text.toUtf8());
         str << "u\"" << pystr.c_str() << "\"";
+        if (it != values.end()-1){
+            str << QLocale().groupSeparator();

```

Comment:
doesn't look right. We should output python code, and group separator is whatever character was set up in OS preferences. I.e, only on some systems it resolves to comma. I think you should put a hard-coded comma instead.